src/Modules/OrchardCore.Commerce.Payment.Stripe/Services/StripePaymentProvider.cs

```diff
@@ -61,11 +61,14 @@ await _session.SaveAsync(new OrderPayment
             });
         }
 
+        var siteSettings = (await _siteService.GetSiteSettingsAsync()).As<StripeApiSettings>();
```

Comment:
Rename this to `stripeApiSettings` or something like that. The `siteSettings` variable name should be reserved for the `ISite` instance.

Reply:
done